internal/config/config.go

```diff
@@ -1,17 +1,29 @@
 package config
 
 import (
+	"fmt"
 	"github.com/kelseyhightower/envconfig"
 )
 
 type specification struct {
-	Port       int    `default:"5000" envconfig:"PORT"`
-	RedisUrl   string `default:"redis://localhost:6379" envconfig:"REDIS_URL"`
-	EnableHsts bool   `split_words:"true"`
+	Port        int    `default:"5000" envconfig:"PORT"`
+	RedisUrl    string `default:"redis://localhost:6379" envconfig:"REDIS_URL"`
+	EnableHsts  bool   `split_words:"true"`
```

Comment:
is envconfig optional?

Reply:
[yea it is](https://github.com/kelseyhightower/envconfig#struct-tag-support), and I believe I have some tests making sure that the env vars work as expected. Really the only reason I'm allowing `PORT` and `REDIS_URL` is because these env vars are used by heroku (our current hosting platform) and expect the application to look for those. I know other PaaS hosting platforms use `PORT` as well. In general though, I'd encourage people to use the `TMPNOTES_` prefix if they have full control of the environment.